provider/provider_yaml_test.go

```diff
@@ -387,18 +387,28 @@ func TestNodePoolGpuAcceleratorPanic(t *testing.T) {
 		},
 		"response": {
 			"properties": {
```

Comment:
Given that we are testing that we don't panic, we can be pretty lenient on what properties we accept.
```suggestion
			"properties": "*"
```

Reply:
Yeah, nice